backend/adapter_processor/models.py

```diff
@@ -80,10 +85,24 @@ class AdapterInstance(BaseModel):
         default=False,
         db_comment="Is the adapter instance currently being used",
     )
+    shared_to_org = models.BooleanField(
+        default=False,
+        db_comment="Is the adapter shared to entire org",
+    )
+
+    is_friction_less = models.BooleanField(
+        default=False,
+        db_comment="Does the adapter created through frictionless onboarding",
+    )
+    is_usable = models.BooleanField(
+        default=True,
+        db_comment="Is the Adpater Usable",
+    )
```

Comment:
At what criteria do we deem it unusable? It'll be good to add a note / docstring on that

Reply:
Added.